app/employee-app/backend/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "wso2"
+name = "employee_app"
+version = "1.0.0"
+distribution = "2201.8.6"
```

Comment:
is there any possibility user a later version like 2201.10.2?

Reply:
+1

Reply:
As discussed with the team, we've decided to bump it to 2201.8.8 for now, as we have conducted testing for that version.